db/schema.rb

```diff
@@ -217,6 +217,10 @@
     t.datetime "requested_at"
     t.boolean "ial2_strict"
     t.string "issuer"
+    t.datetime "verify_doc_submitted_at"
+    t.datetime "verify_doc_results_at"
+    t.string "verify_doc_results"
```

Comment:
Should results be a jsonb column?

Reply:
i'm not a huge fan of jsonb but I used it for proofing_components in profiles so I should probably stay consistent.  Nice catch.

Reply:
With the goal of using this for reporting, `jsonb` gives some more flexibility around querying/indexing 🙂